autopush/tests/test_z_main.py

```diff
@@ -1,3 +1,9 @@
+"""Test main instantiation
```

Comment:
Will you file an issue to clean up these tests?

Reply:
I got it: https://github.com/mozilla-services/autopush/issues/1106